.circleci/config.yml

```diff
@@ -38,41 +38,68 @@ jobs:
           command: |
             docker login --username $DOCKERHUB_USERNAME --password $DOCKERHUB_PASSWORD
 
-            NAME="<< parameters.path >>"
-            TAG=$(date "+%d-%m-%Y")
+            readonly ORB="<< parameters.path >>"
 
-            docker build --tag "ovotech/${NAME}:${TAG}" \
+            for path in $ORB/executor/Dockerfile*; do
+
+              filename=$(basename $path)
+
+              if [[ "$filename" == "Dockerfile" ]]; then
+                  TAG="latest"
+                  BUILD_TAG="$(date "+%d-%m-%Y")"
+              else
+                  TAG="${filename#"Dockerfile-"}"
```

Comment:
I'm not sure I follow why/how there can be multiple executors which could overwrite TAG & BUILD_TAG depending on alphabetical ordering from `for path in $ORB/executor/Dockerfile*; do ...`

Reply:
For each dockerfile we build and push an image.
- The TAG is based on the name of the Dockerfile ('latest', '0.11' etc), and only pushed for the master branch.
- BUILD_TAG is the same but with a timestamp, only used because you need to specify something when you push.
After pushing we get the repo digest of that image and save it to a file named after the tag.

These variable are only used inside the loop.

Reply:
oh yes I didn't spot the indent... my bad